src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -0,0 +1,124 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+import re
+from knack.util import CLIError
+
+from ._utils import is_valid_connection_string, resolve_resource_group
+from .azconfig.models import QueryFields
+
+
+def validate_tags(namespace):
+    if namespace.tags is not None:
+        if not namespace.tags:
+            return
+        tag_content = namespace.tags.split(',')
```

Comment:
This is incorrect. The CLI does not allow comma-separated lists. You should be using the built-in tags_type and not need to duplicate or recreate any of this logic.

Reply:
Actually this method is not used. Will remove it.